src/java.security.jgss/windows/classes/sun/security/krb5/internal/tools/Ktab.java

```diff
@@ -73,52 +73,76 @@ public class Ktab {
      */
     public static void main(String[] args) {
         Ktab ktab = new Ktab();
+        int exitCode = ktab.run(args);
+        if (exitCode != 0) {
+            System.exit(exitCode);
+        }
+    }
+
+    private static class ExitException extends RuntimeException {
```

Comment:
If we want to use ExitException as internal exception, perhaps it's easier to not extending RuntimeException? This way we can rely on compiler to flag all methods which throw it and handle it accordingly.

Reply:
Yes, it comes with a price. It's up to you then.